tarodb/assets_store.go

```diff
@@ -305,6 +312,38 @@ type ManagedUTXO struct {
 	TapscriptSibling []byte
 }
 
+// AssetWithGroup is a subset of the base asset struct that excludes fields such
+// as the witnesses or script key.
+type AssetWithGroup struct {
```

Comment:
From what I can tell, I think the idea here is to only include an assets's human readable fields. With that in mind, I don't think the struct is named (`AssetWithGroup`) correctly. I think it should be `AssetHumanReadable` or something like that.

I think that an "asset with group" is just an instance of an `Asset` or `AssetHumanReadable` which has non-nil group. Creating a new asset type just to constrain the group field seems overly complicated.

> excludes fields such as the witnesses or script key.

Why are those fields excluded in this struct? How does that relate to the grouped property of an asset?

Reply:
Good point, didn't think about using this type elsewhere later on. 

Script keys and witnesses are excluded since I'd say they aren't human-readable; displaying them probably won't help a reader remember which asset is which. The asset ID at least may be helpful to look up in a Universe or other online resource.